src/ftxui/dom/border.cpp

```diff
@@ -1,6 +1,8 @@
 #include "ftxui/dom/node.hpp"
 #include "ftxui/dom/elements.hpp"
 
+#include <algorithm>
```

Comment:
I don't see why this is needed.
Could you please remove every non necessary includes?

Reply:
It is required though, it might be that std::min/max is accidentally pulled in through some other std include, however that is not the same across implementations so for msvc it is needed.

Reply:
I see! Thanks!